### DISCUSSION TITLE
DVWA on Ubuntu Linux

### DISCUSSION THREAD

**Discussion by CyberSpace7777** [community/community#119739] [Packages]
![dvwa error](https://github.com/community/community/assets/167155125/f6416a42-7207-45cb-b561-d8cf42a91d11)
### Select Topic Area

Question

### Body

Subject: DVWA Issue
Attention: Robin Wood

Background: After having completed DVWA installation successfully on Ubuntu Linux, via instructions at https://github.com/digininja/DVWA?tab=readme-ov-file. However, I am having issues with accessing any of Menus on the left pane, which results in errors. Please see below on a Post, posted on Robin Wood's youtube channel https://www.youtube.com/watch?v=Yzksa_WjnY0:

Me: great tutorial. thank you.

Can you please advise how to fix the error below:
Fatal error: Uncaught mysqli_sql_exception: Connection refused in /var/www/html/DVWA/dvwa/includes/dvwaPage.inc.php:535 Stack trace: #0 /var/www/html/DVWA/dvwa/includes/dvwaPage.inc.php(535): mysqli_connect() #1 /var/www/html/DVWA/vulnerabilities/xss_r/index.php(14): dvwaDatabaseConnect() #2 {main} thrown in /var/www/html/DVWA/dvwa/includes/dvwaPage.inc.php on line 535

Robin: Your database credentials are wrong. Raise a ticket in GitHub giving all the details it requests and I'll try to help.

Hi Robin,
My apologies, I couldn't raised a ticket as I don't have a paid account with GitHub.
May you please assist me if possible?

**Comment by digininja** [comment]
You need to create this issue in the DVWA repo, not in this one, this is for GitHub community discussions.

https://github.com/digininja/DVWA/issues

**Reply by CyberSpace7777** [reply]
Hi Robin,
Thank you for having directed me to the link above to raise ticket
I've raised a ticket, "DVWA Post Installation Issue #623".
May you please direct me again if this is still incorrect.
